### ISSUE TITLE
tinyMCE elements lack accessible names

### ISSUE THREAD

**Issue by kimisgold** [omeka/Omeka#1122]
"Description: On the Configure Plugin page, it was found that some edit fields lack accessible names. When navigating to the “Instructions” or the “Thank you” edit fields, they are announced as ‘Rich text area’ and lack the associated label as well as the component role of 'edit field'. The instructions state to ‘Press alt + 0 for help’; however, when doing so, nothing occurs on the page.  

On the same page users can also open a 'Source Code' modal from the text editor toolbars. The multi-line edit field present on the modal also lacks an accessible name.

Expected Results: Edit fields have clear accessible names that match the associated label for the edit field.  

Actual Results: The “Instructions” or the “Thank you” edit fields are announced as ‘Rich text area’"

(1.3.1 Info and Relationships; Level A; WCAG 2.0)
